src/flux/nylas-api.coffee

```diff
@@ -151,6 +151,9 @@ class NylasAPI
     else if env in ['local']
       @AppID = 'n/a'
       @APIRoot = 'http://localhost:5555'
+    else if env in ['custom']
+      @AppID = NylasEnv.config.get('env.AppID') or 'n/a'
```

Comment:
Oh shoot. I forgot about that. I really should test these kinds of things. I never got around to installing the Sync Engine on my home server, which doesn't have any sort of visualization support.